go.mod

```diff
@@ -155,10 +155,14 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
 
 // TODO: remove when middleware will be implemented
-replace github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.0.1-hooks
+replace github.com/cosmos/ibc-go/v3 => github.com/yihuang/ibc-go/v3 v3.0.0-hooks-1
 
 replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.3
 
 // https://github.com/crypto-org-chain/ethermint/tree/v0.9.x-cronos
 // FIXME make a tag or point to upstream when ready
-replace github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.6.1-0.20211207040748-254df3803d62
+replace github.com/tharsis/ethermint => github.com/yihuang/ethermint v0.6.1-0.20220104041757-42fb5e071a49
+
+replace github.com/cosmos/cosmos-sdk => github.com/yihuang/cosmos-sdk v0.44.6-0.20220104045229-ae4cf76ecfa2
```

Comment:
TODO, backported https://github.com/cosmos/cosmos-sdk/pull/10814 to 0.44.x, update after merged.